library/smarty/plugins/function.amcCollect.php

```diff
@@ -1,42 +0,0 @@
-<?php
```

Comment:
Do these function removals have anything to do with POS codes?  If not, a separate PR may be in order.